server/lib/reporting/printer.ts

```diff
@@ -12,6 +12,7 @@ import * as TimSort from 'timsort';
 import { getConfiguration } from '../get-configuration';
 import { REPORTS_PRIMARY_COLOR} from '../../../common/constants';
 import { getSettingDefaultValue } from '../../../common/services/settings';
+import { try } from 'bluebird';
```

Comment:
Check this import

Reply:
I removed here: https://github.com/wazuh/wazuh-kibana-app/pull/4505/commits/95b9020e4c1d1b238fdcd169532c581fa3524501